en/finding-sorting-and-cleaning-entries/groups.md

```diff
@@ -89,6 +89,10 @@ The union of a group and its subgroups is the logical complement of the intersec
 
 You can create a group for your references about music, and then subgroups about the music styles \(classic, jazz, rock, etc.\). By setting the group "Music" as _union_, when you subsequently add references to a subgroup, they will automatically appear in group "Music" as well \(without additional action\).
 
+### Nesting (sub)groups
+
+You can populate your Group pane by configuring JabRef to use the bibtex source's `keywords = {...},` by clicking the `+` icon and followwing the [previous instructions](groups.md#specified-keywords). You can nest subgroups by using the right chevron `>` ([see here](https://github.com/JabRef/jabref/pull/2703/files#diff-e8f986c28ee8a35397cde5cb1352d4662f62be7085cd7d8856db279f1205245dR17)). You achieve this by editing the `keywords = {...},` bibtex field in the entry source by placing `>` between any two keywords where the left-hand keyword is the parent group and the right-hand keyword will be its sub-group. The library entry will be placed there. Note: when you select `+` to do this, the first delimiter must be the right chevron, and the second must be whichever field separator you have configured (by default a comma `,`).
```

Comment:
I made a little error in the order of the delimiters, they should be the other way around: `the first delimiter must be the field separator you have configured (by default a comma ','), and the second must be the chevron.`

Reply:
I'd suggest submitting another PR to the documentation with a fix. I believe it is more user-friendly than pushing another commit to this branch.

Could you also address 👼 
1. Typo in `followwing`
2. Perhaps copy the example `https://github.com/JabRef/jabref/pull/2703/files#diff-e8f986c28ee8a35397cde5cb1352d4662f62be7085cd7d8856db279f1205245dR17)`, or write it as `A > B > C`, instead of linking the code